GitCommands/Git/GitVersion.cs

```diff
@@ -176,7 +176,11 @@ private static int Compare(GitVersion left, GitVersion right)
 
         public override string ToString()
         {
-            return Full.Replace(".msysgit.0", "").Replace(".msysgit.1", "");
+            return Full
+                .Replace(".msysgit.0", "")
+                .Replace(".msysgit.1", "")
+                .Replace(".windows.0", "")
+                .Replace(".windows.1", "");
```

Comment:
Handle only  `.windows.0` and `.windows.1`? There may be more than 2 very minor versions.